R/install_livy.R

```diff
@@ -0,0 +1,93 @@
+#' Install Livy
+#'
+#' Automatically download and install \href{http://livy.io/}{\samp{livy}}.
+#' \samp{livy} provides a REST API to Spark.
+#'
+#' @param spark_home The path to a Spark installation. The downloaded and
+#'   installed version of \samp{livy} will then be associated with this Spark
+#'   installation. When unset (\samp{NULL}), the value is inferred based on
+#'   the value of \samp{spark_version} supplied.
+#' @param spark_version The version of Spark to use. When unset (\samp{NULL}),
+#'   the value is inferred based on the value of \samp{livy_version} supplied.
+#'   A version of Spark known to be compatible with the requested version of
+#'   \samp{livy} is chosen when possible.
+#' @param livy_version The version of \samp{livy} to be installed.
+#' @export
+livy_install <- function(spark_home    = NULL,
```

Comment:
Users usually set `SPARK_HOME` using `Sys.setenv(SPARK_HOME = "")` or directly in their environment. Therefore, I think we can remove this parameter and use `Sys.getenv("SPARK_HOME")` bellow. For `spark_version`, JJs preference suggestion has been to not name `spark_version` but rather only `version` and we've assumed this through the code so I would also rename this one to `version`.